benches/compress.rs

```diff
@@ -82,6 +82,14 @@ fn bench_dbtext(c: &mut Criterion) {
             b.iter(|| unsafe { compressor.compress_into(&buf, &mut buffer) });
         });
 
+        let decompressor = compressor.decompressor();
+        group.throughput(Throughput::Bytes(buffer.len() as u64));
+        group.bench_function("decompress", |b| {
+            b.iter_with_large_drop(|| {
+                decompressor.decompress(&buffer);
+            });
```

Comment:
A couple of points here:

1. By having the semicolon on line 89, the Vec is not being returned to `iter_with_large_drop`, which means the drop isn't delayed, which means you're effectively just measuring as if you were doing `b.iter()`. However, it's possible that without returning your compiler might completely eliminate the decompress call entirely
2. We want to make sure `buffer` is populated if someone tries to run this benchmark harness with a filter, e.g. `cargo bench --bench compress -- 'decompress'`. To do that, you should insert a call to compress_into within `bench_function`, but outside of the `iter_with_large_drop`